kratos/tests/cpp_tests/geometries/test_nurbs_surface.cpp

```diff
@@ -151,6 +152,37 @@ namespace Testing {
             points, p, q, knot_u, knot_v);
     }
 
+    NurbsSurfaceGeometry<3, PointerVector<NodeType>> GenerateReferenceNodeSurfaceWithWeightVector() {
+        PointerVector<NodeType> points;
+
+        points.push_back(NodeType::Pointer(new NodeType(1, 0, 5, 0)));
```

Comment:
Could use make_intrusive_ptr.

Reply:
good point. this function isn't needed anymore anyways ;)